plugins/vendor/zoom/ZoomDropFolderPlugin/batch/KZoomDropFolderEngine.php

```diff
@@ -589,4 +594,20 @@ function updateDropFolderFile($entryId , $dropFolderFile)
 		$this->dropFolderFileService->update($dropFolderFile->id, $kZoomDropFolderFile);
 		$this->dropFolderFileService->updateStatus($dropFolderFile->id, KalturaDropFolderFileStatus::HANDLED);
 	}
+
+	function shouldDisableExpiredDropFolder($dropFolder)
+	{
+		$accessExpiresIn = isset($dropFolder->accessExpiresIn) ? $dropFolder->accessExpiresIn : null;
+		return $accessExpiresIn && $dropFolder->id && $accessExpiresIn <= time() - self::ONE_DAY*3;

```

Comment:
remove $dropFolder->id and write the condition as true/false
also lets raise 5 days

Reply:
done